packages/sui-studio/bin/helpers/cli.js

```diff
@@ -0,0 +1,40 @@
+
+/**
+ * Calls sui-mono comman with sui-studio default config
+ * @param  {String} command Name of the command
+ * @return {ChildProcess}
+ */
+function callSuiMonoCommand (command) {
+  const BIN_PATH = require.resolve('@schibstedspain/sui-mono/bin/sui-mono')
+  callCommand(BIN_PATH, [ command,
+    ...getSuiMonoOptions()
+  ])
+}
+
+/**
+ * Parses sui-mono config in package.json and returns corresponding CLI options
+ * @return {Array<String>} CLI options
+ */
+function getSuiMonoOptions () {
+  const path = require('path')
+  const pkg = require(path.join(__dirname, '..', '..', 'package.json'))
+  const conf = pkg && pkg.config && pkg.config['sui-mono']
```

Comment:
I thought it was nice to make it taken from config... This way if you update sui-mono with new features, you only have to update the config naturally and not to remember the magic behaind sui-mono execution in sui-studio.